jest.config.js

```diff
@@ -12,10 +12,11 @@ if (pluginsToTransform.length > 0) {
 }
 
 /**
- * TODO check if we can process inline webpack loaders (e.g. as found in https://github.com/datavisyn/tdp_core/blob/develop/src/webpack/_bootstrap.ts)
+ * TODO check if we can process inline webpack loaders (e.g. as found in https://github.com/phovea/phovea_ui/blob/master/src/_bootstrap.ts)
```

Comment:
```suggestion
```

I think this line can be removed. Otherwise revert it, because phovea_ui has been merged into tdp_core and the URL is wrong.